pkg/openai/response.go

```diff
@@ -21,3 +21,18 @@ func (resp *OpenAIResponse) GetAllChoices() []string {
 	}
 	return choices
 }
+
+// GetNChoices returns a list of n choices returned by the AI engine.
+// n is the number of completions requested by the user.
+func (resp *OpenAIResponse) GetNChoices(n int32) ([]string, error) {
+	var choices []string
+	var i int32
+	if len(resp.Choices) > 0 {
```

Comment:
This is a nitpick, but I would re-arrange this function so that the error condition is contained in the `if` block rather than the main logic of the function. This way, you can minimize how much you indent and it makes your code look cleaner overall

Reply:
Updated